app/app/src/data/2029.yml

```
@@ -33,3 +33,10 @@
     - 'background-size: auto 100%'
     # - 'background-position-x: right'
     # - 'background-position-y: 5%'
+  - name: Unlockis
```

Add newline after previous item

```suggestion

  - name: Unlockis
```